app/src/main/java/com/vultisig/wallet/presenter/keysign/JoinKeysignViewModel.kt

```diff
@@ -141,7 +139,7 @@ internal class JoinKeysignViewModel @Inject constructor(
                 qrCodeContent ?: run {
                     throw Exception("Invalid QR code content")
                 }
-                val rawJson = qrCodeContent.decodeBase64Bytes().unzip().toString()
+                val rawJson = qrCodeContent//.decodeBase64Bytes().unzip().toString()
```

Comment:
Why this get comment out? we need to unzip

Reply:
Sorry, removed it for testing with Test Flight iOS version